templates/vm.xml.j2

```diff
@@ -35,7 +35,18 @@
       <driver name='qemu' type='{{ volume.format | default(libvirt_volume_default_format) }}'/>
       {% if volume.type | default(libvirt_volume_default_type) == 'file' %}
       <source file='{{ volume.file_path |default(libvirt_volume_default_images_path) }}/{{ volume.name}}'/>
-      {% else %}
+      {% elif volume.type is defined and volume.type == 'network' %}
```

Comment:
`volume.type | default(libvirt_volume_default_type) == 'network'`

Reply:
I've updated this one